inc/sainexthopgroup.h

```diff
@@ -128,6 +181,27 @@ typedef enum _sai_next_hop_group_member_attr_t
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_WEIGHT,
 
     /**
+     * @brief Preferred role in the protection group
+     *
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_PROTECTION
```

Comment:
@kcudnik, can we add condition on the other object's attribute? It seems it is not possible to do it right now?

---
In reply to: [124805486](https://github.com/opencomputeproject/SAI/pull/532#discussion_r124805486) [](ancestors = 124805486)